### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by jpculp**
*Description of changes:*

This change prepares the repository to release new crates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tough/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
...
+## [0.13.0] - 2023-03-02
+### Breaking Changes
+- Add a lifetime parameter to Transport::fetch and Repository::read_target, thanks @sunshowers [#563]
+- Add Send and Sync to Repository transport [#578]
+
+### Changes
+- Fix all_targets iterator type [#562]
+- Addressed clippy warnings [#544], [#556]
+- Remove minor/patch versions from Cargo.tomls [#573]
+- Bump tokio from 1.18.3 to 1.24.1 [#555]
```
Comment (zmrow) [inline tough/CHANGELOG.md:16]: This PR is missing from the list below!

Comment (jpculp) [reply tough/CHANGELOG.md:16]: Good catch!

**Review by zmrow — APPROVED** [review]
:receipt:

**Comment by cbgbt** [comment]
🚀 Github actions is down, but I sat in a call with @jpculp where we ran these verifications. I'm content to merge these.

**Review by cbgbt — APPROVED** [review]

**Pull request merged**